codespell.py

```diff
@@ -80,14 +81,108 @@ def load_text_file(filename):
     return output
 
 
+def checkWords(spell_checker, words):
```

Comment:
Prefer using Python convection (`lower_case_with_underscores`)?

Reply:
For future reference:
* #37